src/renderers/react/WrapperControls.tsx

```diff
@@ -141,6 +124,54 @@ const Tooltip = styled("div")`
   }
 `;
 
+type VerticalPosition = "top" | "bottom";
+type HorizontalPosition = "left" | "right";
+
+const Actions = styled("div")`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: ${actionSpacing}px;
+  z-index: 11;
```

Comment:
This z-index is needed (in PME at least) to avoid clashing with the PME demo rich text controls.